src/openloco/map/tile.h

```diff
@@ -114,14 +116,42 @@ namespace openloco::map
             return type() == TClass ? (TType*)this : nullptr;
         }
 
+        uint8_t base_z() const { return _base_z; }
+
     public:
         surface_element* as_surface() const { return as<surface_element, element_type::surface>(); }
         station_element* as_station() const { return as<station_element, element_type::station>(); }
         building_element* as_building() const { return as<building_element, element_type::building>(); }
         industry_element* as_industry() const { return as<industry_element, element_type::industry>(); }
+        unk1_element* as_unk1() const { return as<unk1_element, element_type::unk_1>(); }
     };
     static_assert(sizeof(tile_element) == 8);
 
+    enum class surface_slope : uint8_t
+    {
+        FLAT = 0x00,
```

Comment:
Should probably be lowercase. Not sure if it should be an enum, could be the same as `window_flags`

Reply:
yeah i have to switch this to a constexpr